src/carousel.js

```diff
@@ -150,10 +160,7 @@ const Carousel = React.createClass({
 
   render() {
     var self = this;
-    var children = this.getChildren();
-
-    children = this.formatChildren(children);
-
+    var children = React.Children.count(this.props.children) > 1 ? this.formatChildren(this.props.children) : this.props.children;
```

Comment:
this line break displaying one item. Just check carousel when it has only one item